src/assets/RustAsset.js

```diff
@@ -0,0 +1,93 @@
+const url = require('url');
+const path = require('path');
+const toml = require('toml');
+const {exec} = require('child-process-promise');
+
+const fs = require('../utils/fs');
+const JSAsset = require('./JSAsset');
+const localRequire = require('../utils/localRequire');
+
+const rustTarget = `wasm32-unknown-emscripten`;
+
+class RustAsset extends JSAsset {
+  collectDependencies() {
+    // Do nothing. Dependencies are collected by cargo :).
+  }
+
+  async parse(code) {
+    this.invalidateBundle();
+    this.invalidate();
+    this.dependencies.clear();
```

Comment:
why is this necessary?

Reply:
oh my bad, this was me trying to fix the rebuild as mentioned in the description, I'll remove it, thanks